src/ixdat/techniques/ms.py

```diff
@@ -543,7 +543,9 @@ def calc_n_dot_0(self, gas=None, w_cap=None, h_cap=None, l_cap=None, T=None, p=N
             p = self.p
 
         pi = np.pi
-        eta = DYNAMIC_VISCOSITIES[gas]  # dynamic viscosity in [Pa*s]
+        _eta_v = list(DYNAMIC_VISCOSITIES[gas].values())  # list of eta(T) for 'gas'
+        _eta_T = list(DYNAMIC_VISCOSITIES[gas].keys())  # list of Ts for 'gas'
```

Comment:
Hmm. This spawned in interesting thought, which I think I have settled for myself once before but forgotten in the mean time, which is if keys and values are guarantied to come out in the same order and if that order is insertion order (which I think we need to the interpolation). The answer to the first seems to be "yes" no-matter the Python version I have looked at, but the answer to the second seems to be, that it depends on the Python version. Have a look at this link: https://docs.python.org/3.7/library/stdtypes.html#dictionary-view-objects and witch between Python 3.6 (which we still support) and Python 3.7 and notice how the comment in the code example changes from:
```python
>>> # keys and values are iterated over in the same order
```
to 
```python
>>> # keys and values are iterated over in the same order (insertion order)
```

To enforce it you can do something like:
```python
from operator import itemgetter
...
_sorted_eta_pairs = sorted(DYNAMIC_VISCOSITIES[gas].items(), key=itemgetter(0))  # Sorted sequence of (T, eta(T)) pairs for 'gas'
_eta_T, _eta_v = zip(*_sorted_eta_pairs)  # tuple of T's and corresponding tuple of eta(T)'s
```

Reply:
What do you think of alternatively to make the dynamic "constants" a list of list in the dictionary of gasses? 
I represented it as a dictionary in a dictionary for easy read of temperature / eta pairs.

but in ..CONSTANTS it could be changed to
```
DYNAMIC_VISCOSITIES = {
"gas" : [ 
        [T1, eta1],
        [T2, eta2],
    ]
}
```

Then the order is preserved when interpreting

`eta =np.interp(T, DYNAMIC_VISCOSITIES['gas'][:,0], DYNAMIC_VISCOSITIES['gas'][:,1]`

and the easy readabilities is preserved compared to my initial thought of lists in the constants.

Reply:
However, this would require numpy to be imported in the constants, which I do not know if that is bad practice or not

Reply:
I don't consider it a problem to import numpy in constants and I think that is a fine solution on all accounts.